doc/siterc.tex

```diff
@@ -638,13 +638,17 @@ \subsection{[test battery]}
 The minimum set of directives that must be supplied to the batch system on the
 site to initiate jobs for the tests.
 
-\subsubsection[{[[}cylc suite settings{]]}]{[cylc] $\rightarrow$ [UTC mode]}
+\subsection[{[[}cylc suite settings{]]}]{[cylc]}
+
+Default values for entries in the suite.rc.
```

Comment:
*...for entries in the suite.rc [cylc] section.*? (there will be other global items that correspond to other sections of the suite.rc file - e.g. in my authentication branch).